packages/ui/src/components/va-tree-view/VaTreeView.demo.vue

```diff
@@ -1,59 +1,340 @@
 <template>
   <VbDemo>
-    <VbCard>
-      <va-button @click="$refs.treeView.expand()">
-        EXPAND ALL
-      </va-button>
-      <va-button @click="$refs.treeView.collapse()">
-        COLLAPSE ALL
-      </va-button>
-
-      <va-tree-view ref="treeView">
-        <va-tree-category
-          label="Category text"
-          is-open
-        >
-          <va-tree-node>
-            Node text
-          </va-tree-node>
-          <va-tree-category
-            label="Category text"
-            is-open
-          >
-            <va-tree-node>
-              Node text
-            </va-tree-node>
-          </va-tree-category>
-        </va-tree-category>
-      </va-tree-view>
+    <VbCard title="Default">
+      <va-tree-view :nodes="nodes" />
+    </VbCard>
+    <VbCard title="Stateless">
+      <va-tree-view :nodes="nodes" :stateful="false" />
+    </VbCard>
+    <VbCard title="Selectable">
+      <p>Default</p>
+      <va-tree-view :nodes="nodes" selectable />
+      <p>Selection strategy</p>
+      <va-radio
+        v-for="(option, index) in selectionTypeOptions"
+        :key="index"
+        v-model="selectionType"
+        :option="option"
+      />
+      <va-tree-view
+        :nodes="nodes"
+        :selection-type="selectionType"
+        selectable
+      />
+    </VbCard>
+    <VbCard title="Colored checkboxes">
+      <va-color-palette
+        v-model="selectedColor"
+        :palette="colorsPalette"
+      />
+      <va-tree-view
+        :nodes="nodes"
+        selectable
+        :color="selectedColor"
+      />
+    </VbCard>
+    <VbCard title="Selection model">
+      <p>Selection strategy</p>
+      <va-radio
+        v-for="(option, index) in selectionTypeOptions"
+        :key="index"
+        v-model="selectionType"
+        :option="option"
+      />
+      <p>
+        Single selection model for two tree views:
+        <br />
+        {{ JSON.stringify(selectedNodes) }}
+      </p>
+      <va-tree-view
+        v-model:checked="selectedNodes"
+        :nodes="nodes"
+        :selection-type="selectionType"
+        selectable
+      />
+      <va-tree-view
+        v-model:checked="selectedNodes"
+        :selection-type="selectionType"
+        :nodes="nodes"
+        selectable
+      />
+    </VbCard>
+    <VbCard title="valueBy">
+      <p>{{ JSON.stringify(selectedNodesValueBy) }}</p>
+      <br />
+      <va-tree-view
```

Comment:
<img width="270" alt="image" src="https://user-images.githubusercontent.com/5394573/191759913-6fd3b467-af0b-43fb-aa1f-7b1fc2511e10.png">

<img width="303" alt="image" src="https://user-images.githubusercontent.com/5394573/191759966-915c119c-b608-475f-919e-478fbb55f31d.png">

Notice I have different v-model depending on where I click. That should not be the case.